extensions/pom.xml

```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>org.mule</groupId>
             <artifactId>mule-core-tests</artifactId>
-            <version>${project.version}</version>
+            <version>${mule.version}</version>
```

Comment:
We need this because the extensions versions not always are going to be aligned with the Mule version. The same approach was taken for all the modules in this pom -->
[MULE-9055](https://www.mulesoft.org/jira/browse/MULE-9055) (https://github.com/mulesoft/mule/commit/1385d0a0fd4cce70799ed07dffc49d902d6179a4) attacks the same issue.